crypto/src/lokid_key.cpp

```diff
@@ -37,4 +55,21 @@ public_key_t calcPublicKey(const private_key_t& private_key) {
     return publicKey;
 }
 
+
+public_key_t derive_pubkey_x25519(const private_key_t& seckey) {
+
+    public_key_t pubkey;
+    crypto_scalarmult_base(pubkey.data(), seckey.data());
```

Comment:
I also prefer is this was `crypto_scalarmult_curve25519_base` though Jagerman said some of these just wrap into the same `crypto_scalarmult_base` function

But explicitness is nice

Reply:
Updated.